plugins/Wizard/System.cpp

```diff
@@ -119,7 +119,11 @@ void System::setWizardEnabled(bool enabled)
         if (QDir(wizardEnabledPath()).exists()) {
             QDir(wizardEnabledPath()).removeRecursively();
         }
-        QFile::copy(currentFrameworkPath(), wizardEnabledPath());
+        if (!QFile::copy(currentFrameworkPath(), wizardEnabledPath())) {
+          // Make en empty file if framework does not exist
+          QFile f(wizardEnabledPath());
+          f.open(QFile::WriteOnly);
```

Comment:
Please match indentation alignment with the rest of the file for consistency (should be 4 spaces inside if block as seen above this change). :)